src/email/create_user_analytics_dataset.R

```diff
@@ -0,0 +1,186 @@
+"
+Create user analytics data set.
+
+Script queries mailchimp API to get user analytics data and then combines it
+into a tidy data.frame.
+
+This will be run on a GitHub Action (.github/workflows/user_audience_analysis.yml)
+Weekly.
+"
+
+# putting this up here so that we can change to PROD later and only need
+# to update here rather than multiple places.
+FP_USER_INTERESTS = "output/user_research/hdx_signals_user_interests.csv"
+FP_USER_INTERACTIONS = "output/user_research/hdx_signals_user_interactions.csv"
+FP_USER_CONTACTS <- "output/user_research/hdx_signals_user_contacts.csv"
+
+STORAGE_ACCOUNT <- "dev"
```

Comment:
shall we move to prod?

Reply:
sure why not! [02971f6](https://github.com/OCHA-DAP/hdx-signals/pull/253/commits/02971f668d1bce94decad27da780552d4a36f45c)